src/blenderbim/blenderbim/bim/module/layer/operator.py

```diff
@@ -73,7 +73,7 @@ def execute(self, context):
             new.name = attribute.name()
             new.is_null = data[attribute.name()] is None
             new.is_optional = attribute.optional()
-            new.string_value = "" if new.is_null else data[attribute.name()]
+            new.set_value("" if new.is_null else data[attribute.name()])
```

Comment:
Use `import_attributes`

Reply:
This throws an error apparently `import_attributes` doesn't handle `select` attributes. Can i modify it like so ?         
```
if isinstance(data_type, tuple) or data_type in ("entity", "select"):
    callback(attribute.name(), None, data) if callback else None
    continue
```
instead of 
```
if isinstance(data_type, tuple) or data_type == "entity":
    callback(attribute.name(), None, data) if callback else None
    continue
```